search-service/src/main/kotlin/com/egm/stellio/search/temporal/util/TemporalQueryUtils.kt

```diff
@@ -42,21 +43,29 @@ fun composeTemporalEntitiesQueryFromGet(
         requestParams,
         contexts
     ).bind()
-
-    if (inQueryEntities)
+    var withTemporalValues = false
```

Comment:
instead of using these 2 booleans, it will be cleaner to switch to an enum similar to the `AttributeRepresentation` one